MfciPkg/Include/Library/MfciDeviceIdSupportLib.h

```diff
@@ -110,40 +112,22 @@ MfciIdSupportGetOem1 (
   );
 
 /**
- * Get OEM2
- *
- * @param[out] Oem2
- * @param[out] Oem2Size
- *
- * It is the callers responsibility to free the buffer returned using FreePool()
- *
- * @return EFI_STATUS
- */
+  Function that returns the Oem2 string and size upon on return.
+  It is the callers responsibility to free the buffer returned using FreePool().
+
+  @param[out] Oem2      The OEM2 string to be returned.
+  @param[out] Oem2Size  The size of the OEM2 string.
+
+  @retval EFI_SUCCESS           The OEM2 string was successfully returned.
+  @retval EFI_UNSUPPORTED       The function is not supported.
+  @retval EFI_INVALID_PARAMETER The Oem2 is NULL.
+  @retval EFI_OUT_OF_RESOURCES  There is not enough memory to allocate the OEM2 string.
+**/
 EFI_STATUS
 EFIAPI
 MfciIdSupportGetOem2 (
   OUT CHAR16  **Oem2,
   OUT UINTN   *Oem2Size  OPTIONAL
   );
 
-/**
- * the following helps iterate over the functions and set the corresponding target variable names
- */
-
-// define a structure that pairs up the function pointer with the UEFI variable name
-typedef struct {
-  MFCI_DEVICE_ID_FN    DeviceIdFn;
-  CHAR16               *DeviceIdVarName;
-} MFCI_DEVICE_ID_FN_TO_VAR_NAME_MAP;
-
-// populate the array of structures that pair up the functions with variable names
-#define MFCI_TARGET_VAR_COUNT  5
-STATIC CONST MFCI_DEVICE_ID_FN_TO_VAR_NAME_MAP  gDeviceIdFnToTargetVarNameMap[MFCI_TARGET_VAR_COUNT] = {

```

Comment:
gPolicyBlobFieldName and gPolicyTargetFieldVarNames have the same pattern

Reply:
thanks, let me do another search on it.

Reply:
Fixed those.